layouts/custompage/downloads.html

```diff
@@ -35,35 +37,19 @@ <h1>Download</h1>
      <tr>
```

Comment:
```xml
    <tr>
      <th>Version</th>
      <th>Release date</th>
      <th>Source download</th>
      <th>Binary download</th>
      <th>Release notes</th>
      <th>Supported</th>
    </tr>
```

Should we add a "Supported" column to mark the specific version whether is "white_check_mark" ?

Reply:
Initially I did that, but then removed unsupported releases, so the column is not needed.

Reply:
So all the Ozone releases version in this table are "white_check_mark"?, but there are a lot of Ozone releases version in this table, but in `SECURITY.md`, and only 1.4.x are "white_check_mark".

Reply:
With this change, only 1.4.0 is listed.

![Screenshot from 2024-11-10 14-55-26](https://github.com/user-attachments/assets/1e545454-f344-408f-a3e7-fa6ff78071f4)

Reply:
OK, understand.